### ISSUE TITLE
Left-over (?) in /tmp directory

### ISSUE THREAD

**Issue by auzias** [nextcloud/docker#1110]
In the docker-image `nextcloud:latest` (30510f8fb18b) there are several files that remain in `/tmp` directory :

``` bash
#find /tmp -type d
/tmp
/tmp/pear
/tmp/pear/temp
/tmp/pear/cache
/tmp/pear/download
```

I believe thoses are unecessary, left-overs of [`pecl install`](https://github.com/nextcloud/docker/blob/b1825a188ba1918aa30d0b6ac18d0a32ff3ec707/18.0/apache/Dockerfile#L61).

While these files do not takes a lot of storage (~820K), I would to know if they are necessary for the running of nextcloud or if they can be removed. If they can, I'll glady propose a PR.

**Comment by joshtrichards** [comment]
Hi @auzias - Thanks for the report! Fixed in #1458.

Please close this Issue out, unless you're still seeing this behavior. Thanks!

**Comment by joshtrichards** [comment]
Fixed #1458
